Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -4935,22 +4940,22 @@ CVehicleAudioSettingsEntry& CClientVehicle::GetOrCreateAudioSettings()
 }
 
 
-bool CClientVehicle::GetDummyPosition(eVehicleDummies dummy, CVector& position) const
+bool CClientVehicle::GetDummyPosition(VehicleDummies dummy, CVector& position) const
 {
-    if (dummy >= 0 && dummy < VEHICLE_DUMMY_COUNT)
+    if (dummy >= VehicleDummies::LIGHT_FRONT_MAIN && dummy < VehicleDummies::VEHICLE_DUMMY_COUNT)
     {
-        position = m_dummyPositions[dummy];
+        position = m_dummyPositions[(std::size_t)dummy];
         return true;
     }
 
     return false;
 }
 
-bool CClientVehicle::SetDummyPosition(eVehicleDummies dummy, const CVector& position)
+bool CClientVehicle::SetDummyPosition(VehicleDummies dummy, const CVector& position)
 {
-    if (dummy >= 0 && dummy < VEHICLE_DUMMY_COUNT)
+    if (dummy >= VehicleDummies::LIGHT_FRONT_MAIN && dummy < VehicleDummies::VEHICLE_DUMMY_COUNT)
```

Comment:
`VehicleDummies` is scoped, right? Then this check is no more needed. But you can add an assertation if you want to be 100% sure everything is correct. The same for other places.

Reply:
That's right, I'll take care of it during the refactoring of that file